src/formatDate.js

```diff
@@ -1,14 +1,62 @@
 'use strict';
 
-/**
- * @param {string} date
- * @param {string[]} fromFormat
- * @param {string[]} toFormat
- *
- * @returns {string}
- */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateToArrow = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
The 'split' method is used to decompose the date string into its constituent parts. Ensure that the 'fromFormat' array's last element is indeed a separator to avoid potential issues.